lib/coinbase/wallet.rb

```diff
@@ -459,6 +459,26 @@ def load_seed(file_path)
       "Successfully loaded seed for wallet #{id} from #{file_path}."
     end
 
+    # Creates a new webhook on the current wallet for tracking wallet activity events.
+    #
+    # @param notification_uri [String] The URI to which the webhook notifications will be sent.
+    # @param signature_header [String] (Optional) A header used to sign the webhook request,
+    #   defaulting to an empty string.
+    #
+    # @return [Coinbase::Webhook] The newly created webhook instance.
+    def create_webhook(notification_uri:, signature_header: '')
+      Coinbase::Webhook.create(
+        network_id: Coinbase.normalize_network(network),
+        notification_uri: notification_uri,
+        event_type: Coinbase::Webhook::WALLET_ACTIVITY_EVENT,
+        event_type_filter: {
+          addresses: addresses.map(&:id),
```

Comment:
Can the `addresses` be determined by the backend?

Otherwise when we create a new address, the user would need to update the webhook or the backend would need to manage it anyways